### PR TITLE
feat(model/message_handler): Add time to log message output

### PR DESCRIPTION
**Pull request by jchauncey**
Testing Instructions:

* clone this PR
* `make build push upgrade`
* checkout the corresponding fluentd PR - https://github.com/deis/fluentd/pull/61
* `make build push upgrade` in the fluentd directory
* Deploy an application that you can use to generate log entries with. 
* Use `deis logs` to see new message format.

The log format should look similar to this - 

```
2016-10-18T20:24:06+00:00 app[foo.web.v4]: 2016/10/18 20:24:05 10.240.0.7 GET / - 6334824724549167320
2016-10-18T20:24:07+00:00 app[foo.web.v4]: 2016/10/18 20:24:06 10.240.0.7 GET / - 605394647632969758
2016-10-18T20:24:07+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.240.0.7 GET / - 1443635317331776148
2016-10-18T20:24:08+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.164.2.1 GET / - 894385949183117216
2016-10-18T20:24:08+00:00 app[foo.web.v4]: 2016/10/18 20:24:07 10.240.0.5 GET / - 2775422040480279449
2016-10-18T20:29:03+00:00 deis[controller.INFO]: buildpack type detected. Defaulting to $PORT 5000
2016-10-18T20:29:03+00:00 deis[controller.INFO]: buildpack type detected. Defaulting to $PORT 5000
2016-10-18T20:29:03+00:00 deis[controller.INFO]: scaling Deployment foo-web from 1 to 4 replicas
2016-10-18T20:29:04+00:00 deis[controller.INFO]: adding 30s on to the original 120s timeout to account for the initial delay specified in the liveness / readiness probe
2016-10-18T20:29:04+00:00 deis[controller.INFO]: This deployments overall timeout is 300s - batch timout is 150s and there are 2 batches to deploy with a total of 4 pods
2016-10-18T20:29:14+00:00 deis[controller.INFO]: waited 10s and 4 pods are in service
2016-10-18T20:29:24+00:00 deis[controller.INFO]: waited 20s and 4 pods are in service
2016-10-18T20:29:34+00:00 deis[controller.INFO]: waited 30s and 4 pods are in service
2016-10-18T20:29:38+00:00 deis[controller.INFO]: admin scaled pods web=4
```

### CODE REVIEW

**Comment by kmala** [comment]
@jchauncey i think you need to change the existing e2e test along with this

File: manifests/deis-logger-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```
Comment (vdice) [inline manifests/deis-logger-deployment.yaml]: I believe this needs to be updated with some more fields from the current version existing in the workflow-dev chart for `make upgrade` to work:
```
 kind: Deployment
 metadata:
   name: deis-logger
+  namespace: deis
   labels:
     heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: canary
 spec:
   replicas: 1
   strategy:
```

File: Makefile [hunk 2]
```diff
@@ -74,7 +74,7 @@ clean: check-docker
 	docker rmi $(IMAGE)
 
 update-manifests:
-	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-logger-rc.yaml > manifests/deis-logger-rc.tmp.yaml
+	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-logger-deployment.yaml > manifests/deis-logger-deployment.tmp.yaml
```
Comment (vdice) [inline Makefile]: we can `git rm manifests/deis-logger-rc.yaml` now, correct?

Comment (jchauncey) [inline Makefile]: yeah i was leaving it around for a bit just in case

File: manifests/deis-logger-deployment.yaml (hunk 1 above)
Comment (jchauncey) [inline manifests/deis-logger-deployment.yaml]: fixed

Comment (vdice) [inline manifests/deis-logger-deployment.yaml]: Unfortunately, I'm still unable to successfully upgrade:
```
sed 's#\(image:\) .*#\1 quay.io/vdice/logger:git-0b6c58a#' manifests/deis-logger-deployment.yaml > manifests/deis-logger-deployment.tmp.yaml
kubectl delete -f manifests/deis-logger-deployment.tmp.yaml
yaml: line 2: mapping values are not allowed in this context
make: *** [kube-update] Error 1
```

Comment (vdice) [inline manifests/deis-logger-deployment.yaml]: @jchauncey d'oh! Looks like it's simply a matter of unintended extra whitespace.  Just need to push the block back one space:
```
- metadata:
-   name: deis-logger
-   namespace: deis
-   labels:
-     heritage: deis
-   annotations:
-     helm-keep: "true"
-     component.deis.io/version: canary
+metadata:
+  name: deis-logger
+  namespace: deis
+  labels:
+    heritage: deis
+  annotations:
+    helm-keep: "true"
+    component.deis.io/version: canary
```

**Pull request merged**